public/components/FeatureAnywhereContextMenu/EnhancedAccordion/EnhancedAccordion.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  EuiTitle,
+  EuiSpacer,
+  EuiButtonIcon,
+  EuiButtonEmpty,
+  EuiAccordion,
+  EuiPanel,
+} from '@elastic/eui';
+import './styles.scss';
+
+const EnhancedAccordion = ({
+  id,
+  title,
+  subTitle,
+  isOpen,
+  onToggle,
+  children,
+  isButton,
+  iconType,
+  extraAction,
+}) => (
+  <div className="euiPanel euiPanel--borderRadiusMedium euiPanel--plain euiPanel--hasShadow euiPanel--hasBorder euiPanel--flexGrowZero euiSplitPanel euiSplitPanel--row euiCheckableCard">
+    <div className="euiPanel euiPanel--paddingMedium euiPanel--borderRadiusNone euiPanel--subdued euiPanel--noShadow euiPanel--noBorder euiPanel--flexGrowZero euiSplitPanel__inner">
+      <EuiButtonIcon
+        color="text"
+        onClick={onToggle}
+        iconType="arrowRight"
+        aria-label="Expand"
+        className={`enhanced-accordion__arrow ${isOpen ? 'enhanced-accordion__arrow--open' : ''} ${
+          isButton ? 'enhanced-accordion__arrow--hidden' : ''
+        }`}
+      />
+    </div>
+    <div className="enhanced-accordion__title-panel euiPanel euiPanel--borderRadiusNone euiPanel--transparent euiPanel--noShadow euiPanel--noBorder euiSplitPanel__inner">
+      {!isButton && (
+        <EuiAccordion
+          id={id}
+          arrowDisplay="none"
+          extraAction={<div className="enhanced-accordion__extra">{extraAction}</div>}
+          forceState={isOpen ? 'open' : 'closed'}
+          onToggle={onToggle}
+          buttonContent={
+            <div className="enhanced-accordion__title">
+              <EuiTitle
+                size="s"
+                onClick={onToggle}
+                role="button"
```

Comment:
what does `role="button"` do here?

Reply:
in order to make it as similar as the mockup, we use an accordion as a button